extensions/sql-migration/src/dialog/assessment/assessmentUtils.ts

```diff
@@ -0,0 +1,216 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { MigrationTargetType } from '../../api/utils';
+import { IssueCategory } from '../../constants/helper';
+import { Page, SavedInfo } from '../../models/stateMachine';
+
+export function parseAssessmentReport(assessmentReport: any): any {
```

Comment:
unless any is needed, we should use specific type. As this the main feature of Typescript to type check with any we won't get that. that.

Reply:
Here assessmentReport is parsed from JSON.

const assessmentReport = JSON.parse(assessmentReportJson)

these has no predefined type thus used "any".